browser/src/control/jsdialog/Widget.TreeView.js

```diff
@@ -420,6 +429,11 @@ function _hasIcon(columns) {
 function _createSelectionElement (parentContainer, treeViewData, entry, builder) {
 	var selectionElement;
 	var checkboxtype = treeViewData.checkboxtype;
+
+	if(entry.image) {
```

Comment:
Please not we use `_createSelectionElement` only when `entry.state` is used in the JSON.
But it can be also used in other cases without `entry.state`. Better to put it before  `_createSelectionElement` in the parent function.